README-deploy-benchmark.md

```diff
@@ -0,0 +1,221 @@
+  export MODEL_DIR=/mnt/models
+  ```
+
+  Once the models are downloaded, you can consider the following methods for sharing them across nodes:
+
+  - Persistent Volume Claim (PVC): This is the recommended approach for production setups. For more details on using PVC, refer to [PVC](https://github.com/opea-project/GenAIInfra/blob/main/helm-charts/README.md#using-persistent-volume).
+  - Local Host Path: For simpler testing, ensure that each node involved in the deployment follows the steps above to locally prepare the models. After preparing the models, use `--set global.modelUseHostPath=${MODELDIR}` in the deployment command.
+
+- Label Nodes
+  ```base
+  python deploy.py --add-label --num-nodes 2
+  ```
+
+### Deployment Scenarios
+
+This deployment supports scalable operations for the following AI examples. You can specify the example name using the `--chart-name` flag:
+
+- **chatqna**
+- **codegen**
+- **codetrans**
+- **faqgen**
+- **docsum**
+
+Note that only **chatqna** has `--with-rerank` support and can run with `--tuned` mode currently.
+
+The example below are based on a two-node setup. You can adjust the number of nodes by using the `--num-nodes` option.
+
+By default, these commands use the `default` namespace. To specify a different namespace, use the `--namespace` flag with deploy, uninstall, and kubernetes command. Additionally, update the `namespace` field in `benchmark.yaml` before running the benchmark test.
+
+For additional configuration options, run `python deploy.py --help`
+
+#### Case 1: Baseline Deployment for Chatqna with Rerank
+
+Deploy Command (with node number, Hugging Face token, model directory specified):
+
+```bash
+python deploy.py --chart-name chatqna --hf-token $HF_TOKEN --model-dir $MODEL_DIR --num-nodes 2 --with-rerank
+```
+
+Uninstall Command:
+
+```bash
+python deploy.py --chart-name chatqna --uninstall
+```
+
+#### Case 2: Baseline Deployment for Chatqna without Rerank
+
+```bash
+python deploy.py --chart-name chatqna --hf-token $HFTOKEN --model-dir $MODELDIR --num-nodes 2
+```
+
+#### Case 3: Tuned Deployment for Chatqna with Rerank
+
+```bash
+python deploy.py --chart-name chatqna --hf-token $HFTOKEN --model-dir $MODELDIR --num-nodes 2 --with-rerank --tuned
+```
+
+#### Case 4: Tuned Deployment for codegen
+
+```bash
+python deploy.py --chart-name codegen --hf-token $HFTOKEN --model-dir $MODELDIR --num-nodes 2
+```
+
+## Benchmark
+
+### Test Configurations
+
+| Key      | Value   |
+| -------- | ------- |
+| Workload | ChatQnA |
+| Tag      | V1.1    |
+
+Models configuration
+| Key | Value |
+| ---------- | ------------------ |
+| Embedding | BAAI/bge-base-en-v1.5 |
+| Reranking | BAAI/bge-reranker-base |
+| Inference | Intel/neural-chat-7b-v3-3 |
+
+Benchmark parameters
+| Key | Value |
+| ---------- | ------------------ |
+| LLM input tokens | 1024 |
+| LLM output tokens | 128 |
+
+Number of test requests for different scheduled node number:
+| Node count | Concurrency | Query number |
```

Comment:
I believe you wanted to say "instance" instead of node.
One might have more than one instance of a service per node.